spec/support/libraryStubData.js

```diff
@@ -4,25 +4,99 @@ var libraryStubData = {
     'bid-transformer.js': function (config) {
         return {
             apply: function (price) {
-                return price;
+                switch (config.roundingType) {
+                    case 'FLOOR':
+                        return Math.floor(price);
+                    case 'NONE':
+                    default:
+                        return Math.round(price);
+                }
             }
         }
     },
     'browser.js': {
```

Comment:
The following functions are natively supported in browser.js:
`getScreenHeight()`, `getScreenWidth()`

The following variable is accessible via browser:
`topWindow` 

The following functions does not exist in browser.js and should be removed:
`getDomain, getTrackingInfo, getTopWindowLocation, getTopWindowUrl, getTopWindowReferrer, isSecure`

Reply:
The current implementation of getScreenHeight and getScreenWidth on browser.js return a hardcoded value. Is that how it should be?

Reply:
In the run time those two functions will return the actual screen height/width. The current implementation is a mock library that demonstrate the function signatures.